python/py-trustme/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-trustme
+version             0.5.2
+categories-append   devel
+platforms           darwin
+supported_archs     noarch
+license             {Apache-2 MIT}
+
+python.versions     27 35 36 37
+
+maintainers         @jandemter {demter.de:jan} openmaintainer
+
+description         #1 quality TLS certs while you wait, for the discerning tester
+long_description    trustme is a tiny Python package that does one thing: it \
+                    gives you a fake certificate authority (CA) that you can \
+                    use to generate fake TLS certs to use in your tests.
+
+homepage            https://github.com/python-trio/trustme/
+master_sites        pypi:t/trustme
+
+distname            trustme-${version}
+
+checksums           rmd160  1c5bc086145a8c0c51f2aa71250e5ff25412314f \
+                    sha256  8b804c55c7bcb5186f1f408c9da1e5fda915e6fe0142f4411ea900c380456e80 \
+                    size    21819
+
+if {${name} ne ${subport}} {
+    depends_build-append    port:py${python.version}-setuptools
+    depends_lib-append      port:py${python.version}-cryptography \
+                            port:py${python.version}-idna
```

Comment:
Given that `noarch` is set, shouldn't these dependences be `depends_run` instead?